sentinel fmt gives panic: runtime error: index out of range [1] with length 1

```text
Run sentinel fmt -check=true $(find . -name "*.sentinel")
panic: runtime error: index out of range [1] with length 1
```

See the complete [log](https://github.com/hashicorp/terraform-foundational-policies-library/pull/30/checks?check_run_id=1315266591).

~Does anyone know the solution to this issue?~

Pull request #2 created to resolve this issue.

As an alternative, see https://github.com/hashicorp/terraform-foundational-policies-library/pull/32.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,12 +1,31 @@
-FROM alpine
+FROM docker.io/library/debian:10-slim
 
-ENV SENTINEL_VERSION=0.15.0
+ENV SENTINEL_VERSION=0.16.1 \
+    PATH=/src:$PATH
 
 WORKDIR /src
 
-ENV PATH /src:$PATH
+RUN apt-get update \
+    && apt-get install \
+        --assume-yes \
+        --no-install-recommends \
+        ca-certificates=* \
+        curl=* \
+        unzip=* \
+    && apt-get clean \
+    && rm \
+        --force \
+        --recursive \
+        /var/lib/apt/lists/*
 
-RUN wget https://releases.hashicorp.com/sentinel/${SENTINEL_VERSION}/sentinel_${SENTINEL_VERSION}_linux_amd64.zip
-RUN unzip sentinel_${SENTINEL_VERSION}_linux_amd64.zip
+RUN curl \
+        --show-error \
+        --silent \
+        --request GET \
+        https://releases.hashicorp.com/sentinel/${SENTINEL_VERSION}/sentinel_${SENTINEL_VERSION}_linux_amd64.zip \
+        --output \
+        sentinel_${SENTINEL_VERSION}_linux_amd64.zip \
+    && unzip \
+        sentinel_${SENTINEL_VERSION}_linux_amd64.zip
 
 ENTRYPOINT ["sentinel"]

--- a/README.md
+++ b/README.md
@@ -1,17 +1,37 @@
-# sentinel-simulator Docker Container
-### Usage
-This repository automatically builds containers for using the [`sentinel`](https://www.terraform.io/docs/enterprise/sentinel/index.html) command line program.
+# sentinel-simulator-docker
 
+This project builds the container image for using the [`sentinel`](https://www.terraform.io/docs/enterprise/sentinel/index.html) command line program.
 
-##### `latest` (default)
+## Getting Started
 
-The `latest` version of this container will copy the current stable version of the binary into the container, and set it for use as the default entrypoint. This will be the best option for most uses, and if you are just looking to run the binary from a container.
-
-You can use this version with the following:
 ```shell
-docker run -i -t thrashr888/sentinel-simulator:latest <command>
+docker run \
+    --interactive \
+    --tty \
+    thrashr888/sentinel-simulator:latest \
+    --version
 ```
 
 ```shell
-docker run -i -t -v $(pwd):/app/ -w /app/ thrashr888/sentinel-simulator:latest test
+docker run \
+    --interactive \
+    --tty \
+    --volume $(pwd):/app/ \
+    --workdir /app/ \
+    thrashr888/sentinel-simulator:latest \
+    fmt -check=true $(find . -name "*.sentinel" -type f)
 ```
+
+```shell
+docker run \
+    --interactive \
+    --tty \
+    --volume $(pwd):/app/ \
+    --workdir /app/ \
+    thrashr888/sentinel-simulator:latest \
+    test $(find . -name "*.sentinel" ! -path "*/testdata/*")
+```
+
+## Description
+
+The `latest` version of this container will copy the current stable version of the binary into the container, and set it for use as the default entry point. This will be the best option for most uses, and if you are just looking to run the binary from a container. See all Sentinel releases [here](https://releases.hashicorp.com/sentinel/).

--- a/release.sh
+++ b/release.sh
@@ -1,17 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-if [ -z "$1" ]
+if [ -z "${1}" ]
 then
   echo "Releases a new version of the Sentinel Docker image."
   echo "Example:"
-  echo "    ./release.sh v0.15.0"
+  echo "    ./release.sh v0.16.1"
   exit 1
 fi
 
-SENTINEL_VERSION=$1
+SENTINEL_VERSION=${1}
 
 git add .
-git commit -m"upgrade to $SENTINEL_VERSION"
+git commit --message "Upgrade to ${SENTINEL_VERSION}"
 git tag ${SENTINEL_VERSION}
 git push
 git push origin ${SENTINEL_VERSION}

